### DISCUSSION TITLE
'maxDist' option in calcGrains

### DISCUSSION THREAD

**Discussion by zmichels** [mtex-toolbox/mtex#1020] [Ask Anything] [answered]
Hi all,

Today I noticed the option 'maxDist' in the documentation for the calcGrains function. Based on the description, I assumed it was a value that allowed for limiting how grain boundaries were connected across non-indexed spaces between points during grain boundary construction. However, I cannot find any difference in the results I get when using it, and probably I am using it incorrectly and/or expecting something different from it. Can anyone give me an example (syntax) of how to use this option?

Thanks and cheers,
Zach

**Comment by kilir** [comment, marked as answer]
Hi Zach,

the following works for me (having a limit on how large grains can grow at all):
```
mtexdata mylonite

plot(ebsd)
grains = ebsd.calcGrains('maxDist',50)
hold on
plot(grains.boundary,'linecolor','r')

grains = ebsd.calcGrains('maxDist',500)
hold on
plot(grains.boundary,'linecolor','k')
hold off
```
While testing it, I agree that there are a few cases where it seems to behave a bit erratic.

Cheers,
Rüdiger

**Comment by zmichels** [comment]
Hi,

Thank you. I definitely misunderstood it before. 

Cheers,
Zach
